flamingpy/examples/decoding.py

```diff
@@ -81,7 +81,7 @@
 
     # An integer label for each nodes in the stabilizer and matching graphs.
     # This is useful to identify the nodes in the plots.
-    node_labels = {node: index for index, node in enumerate(G_stabilizer.graph)}
+    node_labels = {node: index for index, node in enumerate(G_stabilizer.nodes())}
```

Comment:
I think enumerate understands that you enumerate nodes if you just put in a networkx graph -- does it have trouble with retworkx graphs?

Reply:
This way, it works for all implementations of the StabilizerGraph class. This is no more limited to networkx.

On fact, this is the main point of the abstract class.